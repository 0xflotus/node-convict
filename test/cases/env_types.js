exports.conf = {
  bool: {
    format: 'boolean',
    env: "BOOL"
  },
  int: {
    format: 'integer',
    env: "INT"
  },
  num: {
    format: 'number',
    env: 'NUM'
  }
};

exports.env = {
  BOOL: true,
  INT: 77,
  NUM: 789.1011
};
