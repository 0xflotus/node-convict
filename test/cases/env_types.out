{
  "bool": true,
  "int": 77,
  "num": 789.1011
}
